From: Tzafrir Cohen <tzafrir.cohen@xorcom.com>
Subject: Don't install xtalk_send

xtalk_send is a helper tool for xtalk, a low-level libary used by some
Xorcom DAHDI userspace tools. It is hardly needed by most people.

--- a/xpp/xtalk/Makefile.am
+++ b/xpp/xtalk/Makefile.am
@@ -21,8 +21,7 @@ USB_NAME		= libusb
 endif
 endif
 
-noinst_PROGRAMS		= xlist_test xusb_test xusb_test_bypath xtalk_test xtalk_raw_test
-sbin_PROGRAMS		= xtalk_send
+noinst_PROGRAMS		= xlist_test xusb_test xusb_test_bypath xtalk_test xtalk_raw_test xtalk_send
 noinst_LTLIBRARIES	= libxtalk.la
 dist_noinst_HEADERS	= \
 	xtalk_base.h	\
@@ -39,8 +38,6 @@ dist_noinst_HEADERS	= \
 	include/xtalk/xtalk_iface.h	\
 	#
 
-man_MANS		= xtalk_send.8
-
 libxtalk_la_CFLAGS	= \
 		$(COMMON_CFLAGS) \
 		-I$(srcdir)/include \
